src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -52,6 +54,37 @@ public SentryTarget(string dsn)
             _client.Tags.Add("culture", Thread.CurrentThread.CurrentCulture.Name);
             _client.Tags.Add("branch", BuildInfo.Branch);
             _client.Tags.Add("version", BuildInfo.Version.ToString());
+
+            _debounce = new SentryDebounce();
+        }
+
+
+        private List<string> GetFingerPrint(LogEventInfo logEvent)
```

Comment:
Why not just return a string? I think you can put the string.Join in this method.